sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -91,9 +92,15 @@ def online_write_batch(
         ],
         progress: Optional[Callable[[int], Any]],
     ) -> None:
-        raise NotImplementedError(
-            "to be implemented in https://jira.expedia.biz/browse/EAPC-7971"
-        )
+        with MilvusConnectionManager(config.online_store):
+            try:
+                entities = self._format_data_for_milvus(data)
```

Comment:
let's use a different term like "rows" instead so that this doesn't get confused with the `Entity` type